### ISSUE TITLE
fails to parse singleton list for label

### ISSUE THREAD

**Issue by azaroth42** [iiif-prezi/iiif-prezi#26]
Thanks to @rlskoeser. Example failing JSON:

```
{
@context: "http://iiif.io/api/presentation/2/context.json",
@id: "https://plum.princeton.edu/concern/scanned_resources/ph415q7581/manifest",
@type: "sc:Manifest",
label: [
"Chto my stroim : Tetradʹ s kartinkami"
],
```

**Comment by azaroth42** [comment]
Should be fixed by #35 (along with a bunch of other crazy listiness)

**Comment by zimeon** [comment]
Fine in current version:

```
simeon@Cider iiif-prezi (master %)> cat tests/testdata/2.0/example/fixtures/1/manifest.json  | perl -pe 's/"Test 1 Manifest: Minimum Required Fields"/["Test 1 Manifest: Minimum Required Fields"]/' > a.json
simeon@Cider iiif-prezi (master %)> diff tests/testdata/2.0/example/fixtures/1/manifest.json a.json
5c5
<   "label": "Test 1 Manifest: Minimum Required Fields", 
---
>   "label": ["Test 1 Manifest: Minimum Required Fields"], 
simeon@Cider iiif-prezi (master %)> python check_remote_manifests.py file:a.json
------
file:a.json
```
